packages/api/test/global.d.ts

```diff
@@ -1 +1,2 @@
-import '@api/test-utils/vitest.matchers';
+// eslint-disable-next-line import/extensions
+import '@api/test-utils/vitest.matchers.ts';
```

Comment:
This looks funky but because the way the `test-utils` package is setup if we don't have this `.ts` extension here then the import for these Vitest matchers don't get properly loaded. @kanadgupta and I went through a whole pairing on session on this and settled on a method of getting it working with a very complicated `exports` object in the `package.json` for that package but that ultimately broke a lot of things downstream where we're loading definitions out of `@api/test-utils/definitions`.

Just adding this `.ts` extension, and the very minor change to the Vitest config to use `../test-utils` instead of `@api/test-utils` is a cleaner option than a flaky package config.

Reply:
so weird my goodness